resources/build.js

```diff
@@ -20,10 +25,13 @@ if (require.main === module) {
     fs.mkdirSync(path.dirname(destPath), { recursive: true });
     if (filepath.endsWith('.js')) {
       const flowBody = '// @flow strict\n' + fs.readFileSync(srcPath, 'utf-8');
-      fs.writeFileSync(destPath + '.flow', flowBody);
+      writeGeneratedFile(destPath + '.flow', flowBody);
```

Comment:
@saihaj Can you please extract prettifying output into separate PR?
It will help us to compare generated files before and after TS migration.